packages/integration_test/lib/_callback_io.dart

```diff
@@ -90,7 +91,7 @@ class IOCallbackManager implements CallbackManager {
     integrationTestChannel.setMethodCallHandler(_onMethodChannelCall);
     final List<int>? rawBytes = await integrationTestChannel.invokeMethod<List<int>>(
       'captureScreenshot',
-      null,
+      <String, dynamic>{'name': screenshot},
```

Comment:
the above `!_usesFlutterImage` if statement could also check if `Platform.isAndroid`